.gitignore

```diff
@@ -11,3 +11,4 @@
 /tool/
 *.swp
 Gemfile.lock
+/.idea/
```

Comment:
What’s this?

Reply:
It's the workspace folder that's generated by Jetbrains products (RubyMine, Intellij, etc).
If you want me to rollback this line, let me know.